ccm_web/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "dev:server": "nodemon --inspect=0.0.0.0:9229 --config nmon_config.json server/app.ts",
     "dev:client": "webpack --watch --config webpack/webpack.dev.ts",
-    "dev": "npm run dev:server & npm run dev:client",
+    "dev": "wait-port ccm_db:3306 && sequelize db:migrate & npm run dev:server & npm run dev:client",
```

Comment:
@ssciolla indeed you are right, npx was redundant and simple library names was sufficient to run the command.